presentation/src/main/java/com/yapp/presentation/ui/member/score/MemberScore.kt

```diff
@@ -64,8 +81,8 @@ fun MemberScore(
             )
         },
         modifier = modifier
```

Comment:
modifier에 체이닝 함수가 하나일 경우에는 줄바꿈을 안하는 것이 가독성 면에서 좋을 것 같습니다!
```suggestion
        modifier = modifier.fillMaxSize(),
```